pkgs/by-name/xp/xppen/package.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, lib
+, fetchzip
+, libusb1
+, autoPatchelfHook
+, qt5
+, libGL
+}:
+stdenv.mkDerivation rec {
```

Doesn't seem to need `rec`

```suggestion
stdenv.mkDerivation {
```